quick_install.sh

```diff
@@ -94,47 +95,47 @@ function __bliman_quick_install() {
 	trap track_last_command DEBUG
 	trap echo_failed_command EXIT
 
-	# Global variables
-	export BLIMAN_HOSTED_URL="https://raw.githubusercontent.com"
-	export BLIMAN_NAMESPACE="Be-Secure"
-	export BLIMAN_REPO_URL="$BLIMAN_HOSTED_URL/$BLIMAN_NAMESPACE/BLIman/main"
-	export BLIMAN_VERSION="0.1.0"
-	export BLIMAN_LAB_URL="$BLIMAN_HOSTED_URL/$BLIMAN_NAMESPACE/BeSLab/main"
-	
-	# export BLIMAN_NATIVE_VERSION="0.2.9"
-	# infer platform
-
-	BLIMAN_PLATFORM="$(infer_platform)"
-	export BLIMAN_PLATFORM
-
-	if [ -z "$BLIMAN_DIR" ]; then
-		BLIMAN_DIR="$HOME/.bliman"
-		BLIMAN_DIR_RAW="$HOME/.bliman"
-	else
-		BLIMAN_DIR_RAW="$BLIMAN_DIR"
-	fi
-	export BLIMAN_DIR
+        if [ -z "$BLIMAN_DIR" ]; then
+                BLIMAN_DIR="$HOME/.bliman"
+                BLIMAN_DIR_RAW="$HOME/.bliman"
+        else
+                BLIMAN_DIR_RAW="$BLIMAN_DIR"
```

Comment:
What is this condition for?

Reply:
Thiswas already present in code it was for the bliman directory path.